plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2320,8 +2320,22 @@ elementgui.block.properties_redstone=Redstone properties
 elementgui.block.redstone_connect=<html>Does redstone connect to this block?<br><small>Power works even if this is unchecked
 elementgui.block.emits_redstone=Does emit redstone?
 elementgui.block.redstone_power=Emitted redstone power:
+elementgui.block.custom_properties_states=Custom block properties
+elementgui.block.custom_properties.add=Add custom property
+elementgui.block.custom_properties.add.message=Enter the name of the custom property to be added:
+elementgui.block.custom_properties.add.title=New block state property
+elementgui.block.custom_properties.add.input=Property name
+elementgui.block.custom_properties.add.error_duplicate=Property with this name is already defined.
+elementgui.block.custom_properties.add.error_duplicate.title=Duplicate property
+elementgui.block.custom_properties.add.error_invalid_values=There should be at least one possible value added.
+elementgui.block.custom_properties.add.error_invalid_values.title=Invalid values
+elementgui.block.custom_property.name=Property name:
+elementgui.block.custom_property.type=Property type:
+elementgui.block.custom_property.values=Property values:
+elementgui.block.custom_property.remove=Remove this property
 elementgui.common.value=Value:
 elementgui.block.error_block_must_have_name=Block must have a name
+elementgui.block.page_states=Block state properties
```

Comment:
I would simplify this to "Block states" to blend in UI better

Reply:
That would be a spoiler and also controversial with what given state of UI could offer.

Reply:
I mean this PR adds block states. Block states are both properties and JSON model selectors.

Current text weirdly stretches the UI

Reply:
> Current text weirdly stretches the UI

That be a valid point, will see.